project-reports/2024/2024-Q3-Hyperledger-Besu.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Besu
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+The Besu project is thriving at the moment. There's growing interest in both public blockchain networks and private chains, with several Layer 2 solutions considering Besu for their infrastructure. Publicity around the work on public/private feature parity and new contributions is also increasing. Community activity on Discord has risen, contributions are up, and existing contributors and maintainers are collaborating more effectively than before. Overall, the project's health is robust. The #besu-enterprise channel has been effective in directing new interest in private networks to the right support teams. Discord support and engagement with maintainers and documentation remain strong.
```

Comment:
Have we seen any inflow regarding the sunset plan? And have the maintainers discussed a plan with LFDT and amongst contributors to address any necessary issues?